### PR TITLE
Prepare for 0.4.1 release

### PR DESCRIPTION
**Pull request by ataulm**
Only the one change - removing the deprecated annotation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+[Version 0.4.1](https://github.com/novoda/gradle-build-properties-plugin/releases/tag/v0.4.1)
+--------------------------
+
+- Remove `@Deprecated` annotation for BuildProperties.file build properties through project properties ([#PR59](https://github.com/novoda/gradle-build-properties-plugin/pull/59)).
```
Comment (mr-archano) [inline CHANGELOG.md]: `BuildProperties.file`?

File: README.md [hunk 2]
```diff
@@ -80,4 +80,3 @@ A problem occurred configuring project ':app'.
 
 For more advanced configurations, please refer to the [advanced usage](docs/advanced-usage.md).
 
-The latest Groovydoc can be found [here](https://novoda.github.io/gradle-build-properties-plugin/docs/0.4/).
```
[the remark this answers is not in the document]
Comment (mr-archano) [reply README.md]: would you be so nice to create an issue for this please?

File: CHANGELOG.md (hunk 1 above)
Comment (ataulm) [reply CHANGELOG.md]: `BuildProperties#file` method? how do you specify method/function within a class

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ataulm) [reply README.md]: https://github.com/novoda/gradle-build-properties-plugin/issues/61

File: CHANGELOG.md (hunk 1 above)
Comment (mr-archano) [reply CHANGELOG.md]: Also it's quite confusing to mention project in there. What about:
```
Remove `@Deprecated` annotation for `BuildProperties.file()` as the IDE highlights wrong `Project.file()` method as deprecated.
```

Comment (ataulm) [reply CHANGELOG.md]: but wait this sentence doesn't make sense anyway :thinking:

Comment (ataulm) [reply CHANGELOG.md]: commit 48e5d2bb81eb61f2750e5014bcafed36debbd40a
Author: Ataul Munim <hello@ataulm.com>
Date:   Tue Apr 17 15:31:52 2018 +0100

    Update changelog to highlight why we're removing deprecation

Comment (ataulm) [reply CHANGELOG.md]: " build properties through project properties " was the remaining copy&paste from the previous changelog :joy:

**Pull request merged**